modules/ui.py

```diff
@@ -489,7 +491,9 @@ def create_ui():
         extra_tabs.__enter__()
 
         with gr.Tab("Generation", id="img2img_generation") as img2img_generation_tab, ResizeHandleRow(equal_height=False):
-            with gr.Column(variant='compact', elem_id="img2img_settings"):
+            with gr.Accordion("Open for Settings", open=False), gr.Column(variant='compact', elem_id="img2img_settings") \

```

Comment:
this from my testing your syntex don;t work

```py
with a(), b() if conduction b():
  pass
```
the conduction  only applys to immediate term 
which makes this equivalent to
```py
with a(), b():
  pass
```

Reply:
I have a fix commit coming